app/vmalert/README.md

```diff
@@ -1050,6 +1050,8 @@ The shortlist of configuration flags is the following:
      Optional OAuth2 scopes to use for -notifier.url. Scopes must be delimited by ';'.
   -remoteWrite.oauth2.tokenUrl string
      Optional OAuth2 tokenURL to use for -notifier.url.
+  -remoteWrite.sendTimeout duration
```

Comment:
Maybe it is better to name the flag like a config field name?

Reply:
I've used same name as VMAgent flag has to keep it consistent(vmagent [doc](https://docs.victoriametrics.com/vmagent.html)):
```
  -remoteWrite.sendTimeout array
     Timeout for sending a single block of data to the corresponding -remoteWrite.url
     Supports array of values separated by comma or specified via multiple flags.
```